### PR TITLE
ECOSYSTEM doc.

### PR DESCRIPTION
**Pull request by jjasghar**
This is a document so we have a "one source of truth" of how large the
TK ecosystem really is. Ideally it _shouldn't_ be too hard to keep up to date.

### CODE REVIEW

File: ECOSYSTEM.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+[oracle]: https://github.com/chef-partners/kitchen-oraclecloud
+[sadpanda]:  https://github.com/chef-partners/kitchen-ssh-cisco
+[sync]:  https://github.com/coderanger/kitchen-sync
+[ansible]: https://github.com/neillturner/kitchen-ansible
+[puppet]: https://github.com/neillturner/kitchen-puppet
+[salt]: https://github.com/kitchen-salt/kitchen-salt
+[dokken]: https://github.com/someara/kitchen-dokken
+[lxc]: https://github.com/chrisroberts/kitchen-lxc
+[nodes]: https://github.com/mwrock/kitchen-nodes
+[vsphere]: https://github.com/mwrock/kitchen-vsphere
```
Comment (mwrock) [inline ECOSYSTEM.md]: This gem is the deprecated driver. The "official" one is bundled with the provisioner at https://github.com/CenturyLinkCloud/chef-provisioning-vsphere

File: ECOSYSTEM.md [hunk 2]
```diff
@@ -71,7 +71,6 @@ The following are community driven plugins.
 [dokken]: https://github.com/someara/kitchen-dokken
 [lxc]: https://github.com/chrisroberts/kitchen-lxc
 [nodes]: https://github.com/mwrock/kitchen-nodes
-[vsphere]: https://github.com/mwrock/kitchen-vsphere
```
Comment (mwrock) [inline ECOSYSTEM.md]: Sorry I meant to say we should leave this in but just change url to https://github.com/CenturyLinkCloud/chef-provisioning-vsphere

**Comment by jjasghar** [comment]
Failure wasn't relevant, so is this good to go?

**Comment by mwrock** [comment]
:+1:

**Comment by cheeseplus** [comment]
+1

**Pull request merged**